src/main/java/com/powsybl/openloadflow/network/impl/PropagatedContingency.java

```diff
@@ -311,7 +343,23 @@ private static Identifiable<?> getIdentifiable(Network network, ContingencyEleme
         return identifiable;
     }
 
+    public static void addBranchIdsConnectedToLostBuses(LfNetwork network, Set<String> busIdsToLose, Set<String> branchIdsToOpen) {
+        // update branches to open connected with buses in contingency. This is an approximation:
+        // these branches are indeed just open at one side.
+        for (String busId : busIdsToLose) {
+            LfBus bus = network.getBusById(busId);
+            if (bus != null && !bus.isSlack()) {
```

Comment:
I would write a warning if we try to lost the slack bus because user has to be warm that what is has requested has not been done

Reply:
I think I have to skip the whole contingency here... what I have done would lead to strange configurations...